individual-styles.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: antiquewhite;
+}
+
+.main-photo {
+  display: flex;
+  justify-content: center;
+}
+
+img {
+  opacity: 0.4;
+}
+
+.main-photo .content {
+  position: absolute;
```

Comment:
I would always be careful with using absolute positioning with percentages. Easily leads to non-responsive designs. Try changing screensizes, and see how it still looks :)!